.github/workflows/ci.yml

```diff
@@ -2,7 +2,8 @@ name: Build/test
 on:
   push:
     branches:
-      - "**"
+      - master
+    pull_request:
```

Comment:
I took this from how we do it in Metals. https://github.com/scalameta/metals/blob/6001b60afb2774c342a73fea4856d42ff5fa5d1e/.github/workflows/ci.yml#L4-L6